File: push-openapi-python-demo/QueryAppPushStat.py
#!/usr/bin/python
#coding=utf-8
import properties
from aliyunsdkpush.request.v20150827 import QueryAppPushStatRequest
from aliyunsdkcore import client

clt = client.AcsClient(properties.accessKeyId,properties.accessKeySecret,properties.regionId)

request = QueryAppPushStatRequest.QueryAppPushStatRequest()
request.set_AppKey(properties.appKey)
request.set_Granularity('DAY')
request.set_StartTime('')
request.set_EndTime('')
result = clt.do_action(request)

print result

File: OpenApi1.0/push-openapi-php-demo/QueryAppPushStat.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
use \Push\Request\V20150827 as Push;
$accessKeyId = "";
$accessSecret = "";
$appKey = 123456;
$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessSecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\QueryAppPushStatRequest();

$request->setAppKey($appKey);
//按天进行查询
$request->setGranularity("DAY");
//查询近七天的数据
$startTime =  gmdate('Y-m-d\TH:i:s\Z', strtotime('-7 day'));
$endTime = gmdate('Y-m-d\TH:i:s\Z', time());
$request->setStartTime($startTime);
$request->setEndTime($endTime);

$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>
